src/frontend/src/icons/JigsawStack/index.tsx

```
@@ -0,0 +1,9 @@
+import React, { forwardRef } from "react";
+import JigsawStackIconSVG from "./JigsawStackIcon";
+
+export const JigsawStackIcon = forwardRef<
+  SVGSVGElement,
+  React.PropsWithChildren<{}>
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)